test/integration/test-errors.js

```diff
@@ -39,13 +42,13 @@ describe.configure().retryOnSaucelabs().run('error page', function() {
       }, () => {
         return new Error('Failed to find errors. HTML\n' +
             fixture.doc.documentElement./*TEST*/innerHTML);
-      });
+      }, timeout);
     });
   });
 
   it.configure().skipFirefox().skipEdge()
       .run('should show the body in error test', () => {
-        return expectBodyToBecomeVisible(fixture.win);
+        return expectBodyToBecomeVisible(fixture.win, timeout);
```

Comment:
Fair point. Just wanted to make sure it's consistent in case we change the value in the future.